modOdyssey.bas

```diff
@@ -293,6 +299,11 @@ Sub MonsterDied(index As Long)
     Next A
 End Sub
 
+Public Function GetOffsetDistance(SrcXO As Long, SrcYO As Long, DestXO As Long, DestYO As Long) As Long
+Dim A As Long, B As Long, C As Long, Dir As Long, XDist As Long, YDist As Long

```

Comment:
Looks like none of these are used.

Reply:
Yeah, these were unused. I was trying to get projectiles to create a straight line towards the target without using sourceX and sourceY